src/DataReduction/POD.jl

```diff
@@ -0,0 +1,65 @@
+function matricize(VoV::Vector{Vector{FT}}) where {FT}
+    Matrix(reduce(hcat,VoV)')
```

Comment:
You usually want `reduce(hcat,VoV)` to get the right snapshot matrix (each column is a snapshot). For context: The POD basis is simply the orthogonalized basis that captures the column space of the best low rank r approximation to the snapshot matrix, i.e., a basis of the r-dimensional subspace that best embeds the time series data given by the snapshots in the Frobenius norm sense.